examples/Makefile

```diff
@@ -54,6 +54,12 @@ get-dotnetcore:
 get-dotnetcore-verify:
 	kubeless function call get-dotnetcore |egrep hello.world
 
+get-bash:
+	kubeless function deploy get-bash --trigger-http --runtime bash4.3 --handler helloget.foo --from-file bash/helloget.sh
+	echo "curl localhost:8080/api/v1/proxy/namespaces/default/services/get-bash/"	
+
+get-bash-verify:
+	kubeless function call get-bash
 
```

Comment:
we will need an example with requirements, as a suggestion we can have an example that installs git and initializes a repository:

```
bash-git-init:
 kubeless function deploy git-init --trigger-http --runtime bash4.3 --handler git.init --from-file bash/git.sh --dependencies bash/git-deps

bash-git-init:
 kubeless function call git-init |egrep Initialized.empty.Git.repository
```

where git.sh is:
```
#!/bin/bash
init() {
  local dir=`mktemp -u`
  echo `git init $dir` 
}
```

Also note that for this examples to be executed during the tests you need to add them to the file `integration-tests.bats`:

```
@test "Test function: get-bash" {
  test_kubeless_function get-bash
}

@test "Test function: bash-git" {
  test_kubeless_function bash-git
}
```